features/device-config/ssh-scripting/src/main/java/org/opennms/features/deviceconfig/sshscripting/impl/SshScriptingServiceImpl.java

```diff
@@ -131,8 +141,26 @@ private SshInteractionImpl(
                         .connect(user, host, port)
                         .verify(timeout)
                         .getSession();
-                try {
 
+                final InetAddress remoteAddress = ((InetSocketAddress) session.getRemoteAddress()).getAddress();
+
+                final String ipAddress;
+
+                if (remoteAddress instanceof Inet4Address) {
+                    ipAddress = System.getProperties().getProperty(TFTP_SERVER_IPV4_ADDRESS_PROPERTY);
```

Comment:
This has to be OSGI config property instead of a system property

Reply:
done